sphinx/tutorial/rose/suites.rst

```diff
@@ -342,27 +344,32 @@ See the :ref:`Cheat Sheet` for more information.
       Running :ref:`command-rose-suite-run` will cause the suite to be
       installed, validated and run.
 
-      The ``--local-install-only`` command line option will cause the suite
-      to be installed (only on your local machine not on any job hosts) and
-      validated but not run.
-
-      Install the suite::
+      Use the ``--local-install-only`` command-line option to install the
+      suite on your local machine and validate it::
 
          rose suite-run --local-install-only
 
-      Inspect the installed suite, you will find it in::
+      Inspect the installed suite, which you will find in::
 
          ~/cylc-run/rose-suite-tutorial
 
-      You shuld find all files contained in the :term:`suite directory` as well
-      as the :term:`run directory` folders ``log``, ``work`` and ``share``.
+      .. tip::
```

Comment:
I think this tip is a little unnecessary as we've already introduced the suite/run directory stuff. How about clarifying the directory in the following paragraph e.g:

```
This is the :term:`run directory`, here you should find all the files ...
```

Reply:
Fair enough, yes, perhaps it is overkill. Will follow your advice to make it a more subtle reminder.